tensorflow/python/ops/image_ops_impl.py

```diff
@@ -788,6 +788,13 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     If `image` was 3-D, a 3-D float Tensor of shape
     `[target_height, target_width, channels]`
 
+Usage Example:
+    '''python
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
+    x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
```

Comment:
Use >>> to make this a doctest, do not import tensorflow, and print the shape after padding